gravitee-apim-portal-webui-next/src/components/drawer/drawer.component.html

```diff
@@ -0,0 +1,30 @@
+<!--
+
+    Copyright (C) 2024 The Gravitee team (http://gravitee.io)
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+            http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+-->
+<div class="drawer-container">
+  <button class="btn_container" (click)="close()">
```

Comment:
This might help match the design of the site a bit more: (we can play around with it together if you'd like)
```suggestion
  <button mat-icon-button class="btn_container" (click)="close()">
```